src/common/base/NebulaKeyUtils.h

```diff
@@ -39,6 +39,8 @@ class NebulaKeyUtils final {
                                EdgeType type, EdgeRanking rank,
                                VertexID dstId, EdgeVersion ts);
 
+    static std::string prefix(PartitionID partId);
```

Comment:
`prefix` is seems too wide, could you rename this method ? 

`partition prefix` will be better one ? :)

Reply:
the prefix is an overload function, so I follow it.